static/app/views/performance/newTraceDetails/traceOurlogs.tsx

```diff
@@ -40,26 +42,27 @@ export function TraceViewLogsDataProvider({
 }
 
 export function TraceViewLogsSection() {
+  const tableData = useLogsPageData();
+  if (!tableData?.logsData || tableData.logsData.data.length === 0) {
+    return null;
+  }
   return (
-    <InterimSection
-      key="logs"
-      type={TraceContextSectionKeys.LOGS}
+    <FoldSection
+      sectionKey={TraceContextSectionKeys.LOGS as string as SectionKey}
```

Comment:
Should we be adding these to `SectionKey`? I noticed there were a few more keys in the PR that were being forced to `SectionKey`

Reply:
Heyo, we are doing a redesign next week where we are removing the content in the foldable sections to tabs, keeping this as is for now.